src/locale/locales/tr/messages.po

```diff
@@ -3111,16 +3111,16 @@ msgstr "Herkes"
 
 #: src/components/WhoCanReply.tsx:67
 msgid "Everybody can reply"
-msgstr ""
+msgstr "Herkes cevaplayabilir"
```

Comment:
It's `Herkes yanıtlayabilir` on Twitter. Should we keep it the same here?

Reply:
Oh, older `reply` translations were also `yanıt`. Let me change it